main_scenes/game_scene.gd

```diff
@@ -55,8 +57,9 @@ func _process(_delta):
 func _get_event() -> String:
 	if GlobalVariables.DungeonSequence.size():
 		return GlobalVariables.DungeonSequence.pop_front()
-	return "healing_room"
-	
+
+	return "healing_room" # TODO Code fails if we don't return anything
```

Comment:
This should maybe be an assert(false) then.

Reply:
@Cmckenzie01